src/go/rpk/pkg/cli/cmd/start.go

```diff
@@ -92,6 +95,20 @@ func NewStartCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 			}
 			config.CheckAndPrintNotice(conf.LicenseKey)
 			env := api.EnvironmentPayload{}
+			if len(seeds) == 0 {
```

Comment:
Doesn't viper know how to merge from various places?

Reply:
Yeah, but the thing is that `seeds` is a `[]string`, while `Config.Redpanda.SeedServers` is a `[]SeedServer`. So we still gotta parse it.